.yarn/patches/@metamask-assets-controllers-npm-44.0.0-c223d56176.patch

```diff
@@ -33,3 +33,59 @@ index e7eaad2cfa8b233c4fd42a51f745233a1cc5c387..b89849c0caf7e5db3b53cf03dd5746b6
  /**
   * The list of currencies that can be supplied as the `vsCurrency` parameter to
   * the `/spot-prices` endpoint, in lowercase form.
+diff --git a/dist/TokensController.cjs b/dist/TokensController.cjs
```

Comment:
I released 44.1.0 that should include both of these fixes

Reply:
ooh nice , so let me bump the assets controller version on this PR

Reply:
done [c0c1117](https://github.com/MetaMask/metamask-extension/pull/28553/commits/c0c1117abaf5fb75930ee4e4bd6a806f2ba3578c)